docs/cloud_orchestrator.md

```diff
@@ -0,0 +1,17 @@
+# Cloud Orchestrator
+
+This page describes about `Cloud Orchestrator`.
+
+## What's Cloud Orchestrator?
+
+Cloud Orchestrator is a web service that hosting virtual machines or containers
```

Comment:
It would be better to inform that use "virtual machines or containers" as hosts because the term host is used below.

Reply:
Accepted the suggestion, but I changed the term from 'use' to 'manage' since CO doesn't do not only just using hosts, but also creating/deleting hosts.